api/schemas/webhook.json

```diff
@@ -15,10 +15,6 @@
             "description": "The HTTP header name that is added to the event POST",
             "type": "string"
         },
-        "api_key_value": {
-            "description": "The value that the HTTP header 'api_key_name' will be set to",
-            "type": "string"
-        },
         "events": {
             "description": "List of event types to receive",
             "type": "array",
```

Comment:
I would like to suggest we add enum to the events array so that we can validate the event supplied is a valid webhook event. I have had people recently trying to register webhooks and have not read the docs correctly and putting incorrect strings in here. I think it would be nice to return a 400 if the event string does not match those "allowed"?

Reply:
Good idea! Added in https://github.com/bbc/tams/pull/142/commits/7eec78b27a67d99511b4dedb2693b0235eb4deb9